webapp/app/js/perf.js

```diff
@@ -187,37 +187,40 @@ perf.factory('isReverseTest', [ function() {
 perf.factory('PhCompare', [ '$q', '$http', 'thServiceDomain', 'PhSeries',
              'math', 'isReverseTest', 'phTimeRanges',
   function($q, $http, thServiceDomain, PhSeries, math, isReverseTest, phTimeRanges) {
-  var getClassName = function(baselineMin, baselineMax, baselineAvg, currentAvg, test) {
-    var range = math.trimFloat((baselineMax - baselineMin) / 2);
-    if (isReverseTest(test)) {
-      if (currentAvg < baselineAvg - range) {
-        if (currentAvg < baselineMin) {
-          return "compare-regression";
-        }
-        // Still more than the min value we got out of the baseline, so we could
-        // be OK still, but there could be a regression here too.
-        return "compare-notsure";
-      }
-      else if (currentAvg > baselineMax) {
-        return "compare-improvement";
-      }
-      return "";
+
+  // Default stddev if we only have one value - 15%. Used at t_test.
+  var STDDEV_DEFAULT_FACTOR = 0.15;
+
+  var DIFF_CARE_MIN = 1.015; // We don't care about less than 1.5% diff
```

Comment:
yes, 1.015 is the ratio below which we don't care. e.g. if old is 1.0 and new is 1.014, then that ratio would be below the threshold, but, which would be the same if if old was 1.014 and new was 1.0 - the ratio would still be below 1.05, and again, we don't care. It's ratio, but for close enough values to 1 it's close enough to percentage.